File: solutions/cpp/raindrops/1/raindrops.cpp
#include "raindrops.h"
using namespace std;
namespace raindrops {
    string convert(int number) {
       string ret = ""; 
       if (number % 3 ==0 ) ret+="Pling";
       if (number % 5 ==0 ) ret+="Plang";
       if (number % 7 ==0 ) ret+="Plong";
       if (ret=="") ret=to_string(number);
       return ret;
    }
}  // namespace raindrops

File: solutions/javascript/raindrops/1/raindrops.js
//
// This is only a SKELETON file for the 'Raindrops' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const convert = (num) => {
  var ret = ''
  if (num % 3 == 0) ret += 'Pling';
  if (num % 5 == 0) ret += 'Plang';
  if (num % 7 == 0) ret += 'Plong';
  if (ret == '') ret += num.toString();
  return ret;
};
